src/main/java/homework2/Computer.java

```diff
@@ -0,0 +1,29 @@
+package homework2;
+
+public class Computer {
+    private String brand;
+    private int screenSize;
+    private int numberOfCores;
+    private String type;
+
+    public Computer(String brand, int screenSize) {
+        this.brand = brand;
+        this.screenSize = screenSize;
+    }
+
+    public Computer(String brand, int screenSize, int numberOfCores, String type) {
+        this.brand = brand;
+        this.screenSize = screenSize;
+        this.numberOfCores = numberOfCores;
+        this.type = type;
+    }
+
+    public int getScreenSize(){
+        return screenSize;
+    }
+
+    public static void main(String[] args) {
+        Computer workComputer = new Computer("MacBook", 15, 5, "laptop");
```

Comment:
Please move these two objects to the ComputerTest class and verify them there (you can add some additional get() methods to check other fields as well). After that you can remove this main() method

Reply:
this is not done yet